### ISSUE TITLE
Is the website current?

### ISSUE THREAD

**Issue by sxdxs** [JakePartusch/fantasy-tools#189]
Was trying to use it, but my url should work, was open in browser, league id was present & public league.

**Comment by bunit** [comment]
Yeah having the same issue over here. Tried all my browsers and same thing.

**Comment by JakePartusch** [comment]
Yeah, sorry. ESPN changed their auth flow to include recaptcha v3 and I haven't been able to find a workaround :/
